documentation/book/appendix_metrics.adoc

```diff
@@ -50,18 +50,31 @@ The following resources are defined:
 * A `ConfigMap` to manage the configuration of Prometheus Server.
 * A `Service` to provide an easy to reference hostname for other services to connect to Prometheus server (such as Grafana).
 
+Prometheus also provides an alerting system through the https://prometheus.io/docs/alerting/alertmanager/[alert manager] component.
+In order to enable alerting, the provided https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{GithubVersion}/metrics/examples/prometheus/alerting-rules.yaml[`alerting-rules.yaml`] YAML file describes a `ConfigMap` resource which defines some sample alerting rules on Kafka and Zookeeper metrics.
```

Comment:
Will this work up and downstream? With the GithubVersion attribute?

Reply:
I don't think so but it should e part of a different work of rewording this section for upstream.